src/communication/xcomponentMessages.ts

```diff
@@ -112,6 +112,9 @@ export class Deserializer {
     }
 
     public decodeServerMessage(b64Data: string): string {
+        if (b64Data === undefined) {
```

Comment:
if b64Data could be undefined, the right type definition should be b64Data?: string